File: Linked Lists/Student.py
from Person import Person
class Student(Person):
    def __init__(self, newName="none", address="none", phone="999-999-9999", year=9999):
        super().__init__(newName, address, phone)
        self.year = year

    def getGraduationYear(self):
        return(self.year)
    
    def setGraduationYear(self, year):
        self.year = year
    
    def __str__(self):
        stringtoReturn = "Name         : " + self.name \
                       + "\naddress      : " + self.address \
                       + "\nphone        : " + self.phone \
                       + "\nyear   : " + self.year
        return(stringtoReturn)
    

File: Lists in Java/Student.java

public class Student extends Person {
    private int graduationYear;

    public Student(String newName, String address, String phone, int year) {
        super(newName, address, phone);
        graduationYear = year;
    }

    public int getGraduationYear() {
        return graduationYear;
    }

    public void setGraduationYear(int year) {
        graduationYear = year;
    }

    @Override
    public String toString() {
        return super.toString() + "\nGraduation Year: " + graduationYear;
    }
}
